packages/react-composites/src/composites/CallWithChatComposite/PreventDismissOnEvent.tsx

```diff
@@ -0,0 +1,20 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
Can we move https://github.com/Azure/communication-ui-library/blob/main/packages/react-components/src/components/utils/common.ts to `packages/acs-ui-common` and reuse it.